### PR TITLE
Fixed the course images tests

### PR DESCRIPTION
**Pull request by imhassantariq**
#### Story Link
[FIX: Python Tests: Course Image issues](https://edlyio.atlassian.net/browse/EDS-239)

#### PR Description

Fixed the Course Image Test

#### Type of change

Please select the options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Change

#### Checklist before merging:

- [x] Squased
- [ ] Reviewd

### CODE REVIEW

File: openedx/core/lib/courses.py [hunk 1]
```diff
@@ -33,12 +33,15 @@ def course_image_url(course, image_key='course_image'):
         url = settings.STATIC_URL + settings.DEFAULT_COURSE_ABOUT_IMAGE_URL
     else:
         loc = StaticContent.compute_location(course.id, getattr(course, image_key))
-        try:
-            AssetManager.find(loc)
-        except NotFoundError:
-            url = '/static/' + settings.DEFAULT_COURSE_ABOUT_IMAGE_URL
-        else:
-            url = StaticContent.serialize_asset_key_with_slash(loc)
+        if getattr(course, image_key).endswith('images_course_image.jpg'):
```
Comment (HamzaIbnFarooq) [inline openedx/core/lib/courses.py]: Are we sure that `course` will always have `image_key`?

**Comment by edtecheco** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: openedx/core/lib/courses.py (hunk 1 above)
Comment (imhassantariq) [reply openedx/core/lib/courses.py]: In this case yes we are sure. As you can see the above check's we are reaching here after checking if the course has attr.

**Comment by edtecheco** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by edtecheco** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Pull request merged**